packages/eslint-plugin-next/lib/index.js

```diff
@@ -38,6 +39,7 @@ module.exports = {
         '@next/next/no-document-import-in-page': 2,
         '@next/next/no-head-import-in-document': 2,
         '@next/next/no-script-in-document': 2,
+        '@next/next/no-style-in-document': 2,
```

Comment:
The name should probably be different, as mentioned, not every type of `<style>` is disallowed. This name change should be reflected in the file name, and the rule description etc. as well.

Reply:
Updated rule name and file names as well

Reply:
only this actual line was forgotten to be renamed 😄

Reply:
> only this actual line was forgotten to be renamed 😄

Sorry! renamed.